### ISSUE TITLE
含有某些标签的图片 下载全部失败

### ISSUE THREAD

**Issue by 2635384122** [usaginya/MoeLoader-Delta#123]
最近只能查看图片,站点下载图片都会失败,更换任何节点都是一样.梯子客户端是v2rayN-core3.29和4.13两个版本. p站可以下载,像yande和sankaku下载不了

**Comment by 2635384122** [comment]
今天突然又能下载了,,,不知道之后会怎么样

**Comment by kenji5575** [comment]
搜寻字词最后面有空格的话,虽然搜寻得到图,但是下载会失败

看看是不是遇到这问题

**Comment by 2635384122** [comment]
> 搜寻字词最后面有间隔的话，虽然搜寻得到图，但是下载会失败
> 
> 看看是不是遇到这问题

不是,我搜的一个单词.  我发现好像是含有某些标签的图片无法下载,比如dakimakura搜索的图片就全都是无法下载,就11天前成功了一次.我一般搜索作者的名字,比如twinbox,可以下载,但是下载速度达不到峰值很慢容易失败,大部分重试可以成功. 此作者有些图片就是下载失败,比如yande 图片ID 750517 ,不知道是受哪个标签影响,无法下载下载进度一直0%. 你可以搜下此作者,下载看看. 我脑血栓已经出来了

**Comment by 2635384122** [comment]
> 搜寻字词最后面有间隔的话，虽然搜寻得到图，但是下载会失败
> 
> 看看是不是遇到这问题
仔细看了下,非常感谢,我去看了下error文件,发现dakimakura是找不到路径,路径里面 \看着好像多个空格,我就去看了下我的关键词,我之前那个确实单个单词后面不知道怎么回事加了个空格......  不过我上面说的像twinbox这个搜索词,没有空格,就是有几张图片无法下载.
图片目录的日志:
2021/4/23 5:46:33
https://files.yande.re/image/a375745dc1a527eecc95e5530e5b7cec/yande.re%20750760%20areola%20bra%20dress%20kodama_haruka%20open_shirt%20pantsu%20stockings%20thighhighs%20twinbox%20twinbox_school.jpg
2021/4/23 5:46:33
E:\批量下载图片\MoeLoader 图站下载\TwinBox\[yande_750760_Radioactive]areola bra dress kodama_haruka open_shirt pantsu stockings thighhighs twinbox twinbox_school_00000.jpg
2021/4/23 5:46:33
未将对象引用设置到对象的实例。

软件目录下的错误日志:
[异常详细]:
System.Net.WebException: 远程服务器返回错误: (400) 错误的请求。
   在 System.Net.HttpWebRequest.GetResponse()
   在 MoeLoaderDelta.SessionClient.Get(String url, IWebProxy proxy, String pageEncoding, SessionHeadersCollection shc)

[异常时间]: 2021/4/23 5:48:39
[异常信息]: SessionClient Get Error
 | 远程服务器返回错误: (400) 错误的请求。
[请求域名]: yande.re
[调用堆栈]: 在 System.Net.HttpWebRequest.GetResponse()
   在 MoeLoaderDelta.SessionClient.Get(String url, IWebProxy proxy, String pageEncoding, SessionHeadersCollection shc)
[触发方法]: System.Net.WebResponse GetResponse()
[请求返回]: nginx - BadRequest - BadRequest
Connection: close
Strict-Transport-Security: max-age=31536000; includeSubDomains; preload
Content-Length: 226
Content-Type: text/html; charset=utf-8
Date: Thu, 22 Apr 2021 21:48:39 GMT
Server: nginx

[异常详细]:
System.Net.WebException: 远程服务器返回错误: (400) 错误的请求。
   在 System.Net.HttpWebRequest.GetResponse()
   在 MoeLoaderDelta.SessionClient.Get(String url, IWebProxy proxy, String pageEncoding, SessionHeadersCollection shc)
